runner.py

```diff
@@ -44,32 +35,10 @@
         help="process only this centre's CSV files",
     )
 
-    parser.set_defaults(once=True)
     parser.set_defaults(sftp=False)
     parser.set_defaults(keep_files=False)
     parser.set_defaults(add_to_dart=False)
 
     args = parser.parse_args()
 
-    if args.once:
-        main.run(
-            sftp=args.sftp, keep_files=args.keep_files, add_to_dart=args.add_to_dart, centre_prefix=args.centre_prefix
-        )
-    else:
-        print("Scheduled to run every 15 minutes")
-
-        # If a run misses its scheduled time, it queues up. If more than one run is queued up, they execute sequentially
-        # i.e. no parallel processing happens
-        schedule.every(15).minutes.do(
-            main.run, sftp=args.sftp, keep_files=args.keep_files, add_to_dart=args.add_to_dart
-        )
-
-        while True:
-            try:
-                schedule.run_pending()
-                time.sleep(1)
-            except Exception as e:
-                logger.error("There was an exception while running the scheduler")
-                logger.exception(e)
-                # We wait 60 seconds so it wont try to check it again during the same minute
-                time.sleep(60)
+    main.run(sftp=args.sftp, keep_files=args.keep_files, add_to_dart=args.add_to_dart, centre_prefix=args.centre_prefix)
```

Comment:
I thought we were going to try the flask command line functionality instead? That way, we wouldn't need to worry about processing any env vars as the flask engine would do that for us.

Reply:
Although, just wondering how to hook into the processing when the lighthouse service requests a "crawl".

Reply:
As discussed, let's not change this as it's a development only concern.